avocado/core/nrunner/task.py

```diff
@@ -47,15 +47,15 @@ def post(self, status):
             host, port = self.uri.split(":")
             port = int(port)
             if self.connection is None:
-                for _ in range(30):
+                for _ in range(600):
```

Comment:
Hi @pevogam can you please give me more context to this change? I don't understand why you need to increase the timeout. Thank you.

Reply:
Hi @richtja at least empirically most containers as well as remote hosts could take much longer than 30 seconds in various situations (hardware load, being booted, etc.). As the above loop will be interrupted earlier if the connection is established earlier it won't slow down faster cases but will give more tolerance to slower cases (I have obtained multiple timeouts here due to 30 seconds being too restrictive).

Reply:
I see, thanks for the explanation.